src/Jackett.Common/Definitions/xtremezone.yml

```diff
@@ -63,54 +64,58 @@
     error:
       - selector: td.embedded:has(center > h2:contains(failed))
     test:
-      path: browse.php
-      
+      path: browse
+
   ratio:
-    path: browse.php
+    path: browse
     selector: font:contains("Ratio:")+font
 
   search:
     paths:
-      - path: browse.php
+      - path: browse
     inputs:
-      $raw: "{{range .Categories}}c{{.}}=1&{{end}}"
+      $raw: "{{range .Categories}}&categories[]={{.}}{{end}}"
```

Comment:
while I can't be sure, since I cannot test this for myself, i think the $raw needs to be 
```yaml
      $raw: "{{range .Categories}}categories[]={{.}}&{{end}}"
```
otherwise the next parm, `search`, will not get prefixed with an amper when categories are in use.

Reply:
I just did some tests, both ways seems to work the same.